Documented that too large messages are accepted and dropped for HTTP API (#6278)
DataDog/documentation: content/en/api/logs/logs.md

* Maximum size for a single log: 256kB
* Maximum array size if sending multiple logs in an array: 500 entries

> All logs exceeding 256kB are accepted and dropped by the platform:

Reviewer: ```suggestion
All logs exceeding 256KB are accepted and dropped by the platform:
```

> * for a single-log request the API drops the log and returns a 2xx

Reviewer: ```suggestion
* For a single log request, the API drops the log and returns a 2xx.
```

> * for a multi-logs request the API processes all the logs smaller or equal to 256kB, drop the others and returns a 2xx

Reviewer: ```suggestion
* For a multi-logs request, the API processes all the logs less than or equal to 256KB, drops the larger logs, and returns a 2xx.
```

* Maximum size for a single log: 256kB
* Maximum array size if sending multiple logs in an array: 500 entries

> All logs exceeding 256kB will be accepted and dropped by the platform.

Reviewer: ```suggestion
All logs exceeding 256KB are accepted and dropped by the platform.
```
